### DISCUSSION TITLE
Ephemeral Specific Port

### DISCUSSION THREAD

**Discussion by vtruong68** [dhruvinsh/ws-ephemeral#2] [Q&A]
Are the ephemeral ports requested compatible with a qbittorrent docker container instance? I was under the impression only matching ports would work, which you aren't allowed to specify ahead of time on Windscribe.

Qbittorrent option:
![image](https://user-images.githubusercontent.com/31712352/190451504-1f9a4c66-29f7-48bc-8bf2-043b4a74f056.png)

In my example, I'd want to request port 19999, but I'd get something like 10512 -> 19999 on Windscribe. Does this still work? Using a [portchecker tool](https://portchecker.co/check) seems to indicate only 10512 is open (19999 is closed) and thus I'm not sure if my qbittorrent container would know that if port 19999 is plugged in.

**Comment by dhruvinsh** [comment]
yes it works perfectly fine.

I am using binhex's [arch-qbittorrentvpn](https://github.com/binhex/arch-qbittorrentvpn) behind Windscribe. And I have similar setup as yours, (unchecked UPnP as well) and it works fine. 

It just take time to show green connection, usually I try to grab something like Ubuntu ISO and it will show green status in qBittorrent at the bottom bar. 

Now as your setup is set for 19999 port, with ws-ephemeral docker, set env variable for the same port and you are good to go.

**Comment by vtruong68** [comment]
I tried the specific port just now with uTP disabled ([according to this post I found about uTP](https://airvpn.org/forums/topic/23159-qbittorrent-connection-icon-green-even-though-port-not-forwarded/)) and it was showing that I was firewalled. Apparently, you get the green status when firewalled just by having uTP enabled. When I switched to a matching port, I got the nice green plug icon right away even with uTP off.
